src/components/circular-progress/examples/circular-progress-props.tsx

```diff
@@ -0,0 +1,48 @@
+import { Component, h } from '@stencil/core';
+/**
+ * Using the props
+ * This component is initially designed to visualize a percentage on a scale of
+ * zero to 100. However, you can easily visualize a progress in other scales,
+ * simply by setting `maxValue` and `suffix`.
+ *
+ * Look at this example to see how the component displays an angel in a
```

Comment:
```suggestion
 * Look at this example to see how the component displays an angle in a
```

Reply:
@Kiarokh Didn't I make this exact comment? 😄

Maybe you shouldn't resolve comments before you've actually fixed them? 😝

Reply:
…or maybe you just hadn't pushed the fix I guess… 🙂

Reply:
I did not see you commented before me. I started reviewing and it took me a while to publish my comment.

Reply:
Ah! 🤦‍♂️ Of course, that makes sense 👍😄